frontend/src/routes/project/[project_code]/+page.svelte

```diff
@@ -1,16 +1,27 @@
 <script lang="ts">
-	import AddProjectUser from '$lib/components/modals/AddProjectUser.svelte';
-import FormatDate from '$lib/components/FormatDate.svelte'
-	import FormatProjectType from '$lib/components/FormatProjectType.svelte'
-	import FormatRetentionPolicy from '$lib/components/FormatRetentionPolicy.svelte'
-	import FormatUserProjectRole from '$lib/components/FormatUserProjectRole.svelte'
-	import HgWeb from '$lib/components/HgWeb.svelte'
-	import t from '$lib/i18n'
-	import type { PageData } from './$types'
-
-	export let data: PageData
-
-	$: project = data.project
+	import AddProjectUser from './AddProjectUser.svelte';
+	import FormatDate from '$lib/components/FormatDate.svelte';
+	import FormatProjectType from '$lib/components/FormatProjectType.svelte';
+	import FormatRetentionPolicy from '$lib/components/FormatRetentionPolicy.svelte';
+	import FormatUserProjectRole from '$lib/components/FormatUserProjectRole.svelte';
+	import HgWeb from '$lib/components/HgWeb.svelte';
+	import t from '$lib/i18n';
+	import type { PageData } from './$types';
+	import DeleteModal from '$lib/components/modals/DeleteModal.svelte';
+	import { _deleteProjectUser, type ProjectUser } from './+page';
```

Comment:
now I get why you were exporting those things... the more appropriate place for those would be in a separate module if you're going to share it or just do your thing in the `deleteProjectUser` on L19... you're extending `+page.ts`'s intended purpose which is to simply deal with data for the page. see (https://kit.svelte.dev/docs/load#page-data)

Reply:
I would do it in the page.svelte because it doesn't need to be shared (for the moment), but I can't because I can't put gql in `.svelte` files and still get type safety. So where else should I put it?

Reply:
you can still drop it in a local module, i.e., a `.ts` file, even if you don't share it, you're sort of forced to get it out because of the gql... this might drive you to consider a clean api that simplly accepts parameters and converts those to gql   (something similar to the way  https://github.com/silinternational/wecarry-ui/blob/cc9146d0cb4c890161cd5230b4a8cc733e1a02e0/src/data/messaging.js calls https://github.com/silinternational/wecarry-ui/blob/cc9146d0cb4c890161cd5230b4a8cc733e1a02e0/src/data/gqlQueries.js  perhaps)or encapsulates domain, e.g., a user module that encapsulates all the user functionality (similar to something like this: https://github.com/silinternational/wecarry-ui/blob/develop/src/data/user.js).